settings.gradle

```diff
@@ -1,4 +1,4 @@
-rootProject.name = 'okhttp-parent'
+rootProject.name = "🍓 okhttp"
```

Comment:
open to suggestions on alternatives!

Reply:
You're creating an arms race now. How about just OkHttp? You can type and filter the projects so it's only one or two characters away as is.

Reply:
Not in the ‘open recent’ menu?

Reply:
Probably okhttp, moving okhttp (directory) to okhttp-core and then reverting the artifactId in publishedArtifactId back to okhttp.